optex/base/graphics.opm

```diff
@@ -59,6 +59,11 @@
   \_def\rotatebox#1#2{\_pdfrotate{#1}#2}%
   \_def\lineheight#1{}%
   \_def\setlength#1#2{}%
+  \_def\transparent#1{}%
```

Comment:
I don't know what `\tranasparent` does mean in LaTeX, maybe: `\def\transparent#1{{\_transparency255 #1}}` ?